ethicalads-theme/templates/ea/homepage.html

```diff
@@ -185,7 +184,7 @@ <h2 class="font-weight-bold mb-8">We understand developers</h2>
                 <h4 class="font-weight-bold">Market to developers</h4>
 
                 <!-- Text -->
-                <p class="text-gray-800 mb-0">Reach millions of developers instantly on our advertising network. We allow targeting by topics and geography, so that you reach the exact audience that you want. Want to reach Data Scientists in the US? We can do that.</p>
+                <p class="text-gray-800 mb-0">Our developer ad network is transparent and keeps your brand safe by showing you exactly where your ads run and how they're performing in near real time.</p>
```

Comment:
Another long sentence:

```suggestion
                <p class="text-gray-800 mb-0">Our developer ad network is transparent and <strong>keeps your brand safe</strong>. See exactly where your ads run and how they're performing in near real time.</p>
```